docs/node/1-node-setup.md

```diff
@@ -0,0 +1,113 @@
+sudo apt -qy upgrade
+```
+## Install Go
+AtomOne requires Go for building the binaries. We will install the Go programming language and set the appropriate environment variables.
+```
+sudo rm -rf /usr/local/go
+curl -Ls https://go.dev/dl/go1.22.10.linux-amd64.tar.gz | sudo tar -xzf - -C /usr/local
+eval $(echo 'export PATH=$PATH:/usr/local/go/bin' | sudo tee /etc/profile.d/golang.sh)
+eval $(echo 'export PATH=$PATH:$HOME/go/bin' | tee -a $HOME/.profile)
+```
+## Download and build binaries
+Next, we will clone the AtomOne project repository, check out the specific version, and build the binaries required for running the node.
+```
+# Clone project repository
+cd $HOME
+rm -rf atomone
+git clone https://github.com/atomone-hub/atomone
+cd atomone
+git checkout v1.1.2
+
+# Build binaries
+make build
+
+# Prepare binaries for Cosmovisor
+mkdir -p $HOME/.atomone/cosmovisor/genesis/bin
+mv build/atomoned $HOME/.atomone/cosmovisor/genesis/bin/
+rm -rf build
+
+# Create application symlinks
+ln -s $HOME/.atomone/cosmovisor/genesis $HOME/.atomone/cosmovisor/current -f
+sudo ln -s $HOME/.atomone/cosmovisor/current/bin/atomoned /usr/local/bin/atomoned -f
+```
+## Install Cosmovisor and create a service
+Cosmovisor is a tool used to manage upgrades for blockchains. Here, we install Cosmovisor and create a systemd service for the AtomOne node so it can start automatically and run in the background.
+```
+# Download and install Cosmovisor
+go install cosmossdk.io/tools/cosmovisor/cmd/cosmovisor@v1.6.0
+
+# Create service
+sudo tee /etc/systemd/system/atomone.service > /dev/null << EOF
+[Unit]
+Description=atomone node service
+After=network-online.target
+
+[Service]
+User=$USER
+ExecStart=$(which cosmovisor) run start
+Restart=on-failure
+RestartSec=10
+LimitNOFILE=65535
+Environment="DAEMON_HOME=$HOME/.atomone"
+Environment="DAEMON_NAME=atomoned"
+Environment="UNSAFE_SKIP_BACKUP=true"
+Environment="PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:$HOME/.atomone/cosmovisor/current/bin"
+
+[Install]
+WantedBy=multi-user.target
+EOF
+sudo systemctl daemon-reload
+sudo systemctl enable atomone.service
+```
+## Set node configuration
+Now, let’s configure your AtomOne node. We need to set the chain ID, keyring backend, and the node's address for communication.
+```
+# Set node configuration
+atomoned config set client chain-id atomone-1
+atomoned config set client keyring-backend os
+atomoned config set client node tcp://localhost:26657
+```
+
+## Initialize the node
+We will initialize the node, download the genesis file, configure the address book, and add some peer nodes to connect to the network.
+```
+# Initialize the node
+atomoned init $MONIKER --chain-id atomone-1
+
+# Download genesis and addrbook
+curl -Ls https://ss.nodeist.net/atomone/genesis.json > $HOME/.atomone/config/genesis.json
+curl -Ls https://ss.nodeist.net/atomone/addrbook.json > $HOME/.atomone/config/addrbook.json
+
+# Add peers
```

Comment:
Maybe not advise people to use peers, seeds are a better practice, and are more stable on long term.

it could be replace with:
```bash
SEEDS=$(curl -s https://chains.cosmos.directory/atomone | jq -r '[.chain.peers.seeds[] | .id + "@" + .address] | join(",")' )
```